webpack/scenes/AlternateContentSources/Create/Steps/SelectSource.js

```diff
@@ -38,25 +53,41 @@ const SelectSource = () => {
           isRequired
         >
           <Flex>
-            <FlexItem>
```

Comment:
This layout makes it look good.
<Grid hasGutter>
           <GridItem span={4} rowSpan={2}>